src/interactivity/win32/ut_interactivity_win32/UiaTextRangeTests.cpp

```diff
@@ -695,6 +695,30 @@ class UiaTextRangeTests
                     {0, bottomRow}
                 }
             }
+
+            MoveTest{
+                L"can move to a new row when necessary when moving forward",
+                { lastColumnIndex, 0 },
+                { lastColumnIndex, 0 },
+                5,
+                {
+                    5,
+                    {lastColumnIndex , 0 + 5},
+                    {lastColumnIndex , 0 + 5}
+                }
+            },
+
+            MoveTest{
+                L"can move to a new row when necessary when moving backward",
+                { 0, 0 + 1 },
+                { lastColumnIndex, 0 + 1 },
+                -5,
+                {
+                    -5,
+                    {lastColumnIndex - 5, 0},
+                    {lastColumnIndex - 5, 0}

```

Comment:
Subtracted the 5 to the x-value instead of the y-value.
Also, keep in mind for when you address the last comment, you'll be starting at a new y-value. So instead of `0-5` it'll be `7-5` (if you choose to make start/end begin on the 7th line, for example).